pyproject.toml

```diff
@@ -254,11 +254,13 @@ show_error_codes = true
 exclude = ["docs", "test", "tests"]
 
 [[tool.mypy.overrides]]
-module = [
-    "airbyte_protocol",
-    "airbyte_protocol.models"
-]
-ignore_missing_imports = true  # No stubs yet (😢)
+module = ["airbyte_protocol", "airbyte_protocol.models"]
+ignore_missing_imports = true                            # No stubs yet (😢)
+
+[tool.pyright]
+pythonVersion = "3.9"
+venvPath = "./"       # Assuming .venv is at the root of your project
+venv = ".venv"
```

Comment:
Looks good!

FYI, I have started adding https://github.com/airbytehq/PyAirbyte/blob/main/poetry.toml to my projects (including this one), as it ensures a deterministic venv directory. So, this is safe for Pyright and also other processes or documentation that requires locating the venv.